css/index.css

```diff
@@ -1,3 +1,559 @@
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+    margin: 0;
+    padding: 0;
+    border: 0;
+    font-size: 100%;
+    font: inherit;
+    vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+    display: block;
+}
+body {
+    line-height: 1;
+}
+ol, ul {
+    list-style: none;
+}
+blockquote, q {
+    quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+    content: '';
+    content: none;
+}
+table {
+    border-collapse: collapse;
+    border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+* {
+    box-sizing: border-box;
+/*    background-color: rebeccapurple;
+*/}
+
+html {
+    font-size: 62.5%;
+}
+
+
+.container {
+    font-size: 1.6rem;
+    width: 100%;
+    max-width: 880px;
+    margin: 0 auto;
+    border: 8px solid orange;
+}
+
+.navigation {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+
+nav {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    border: 5px solid red;
+}
+
+nav a {
+    text-decoration: none;
+    font-size: 1.4rem;
+    color: gray;
+}
+
+.logo {
+    border: 3px solid lime;
+}
+
+
+
+.header-img, .middle-img {
+   
+    width: 100%
+}
+
+/*.logo {
+    margin-left: 20px;
+    border: 3px solid purple;
+    */
+}
+
+
+
+h1 {
+    display: flex; 
+    width: 100%;
+}
+
+.blockone {
+    padding: 60px;
+    font-size: 6rem;
+    text-align: center;
+    border: 3px solid orange;
+}
+
+.blocktwo {
+    border: 3px solid green;
+
+}
+
+header {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    margin-top: 70px;
+    margin-bottom: 50px;
+    border-bottom: 2px solid black;
+
+}
+
+/*h2 {
+    font-size: 1.5rem;
+}
+
+p {
+    font-size: 1.5rem;
+}*/
+
+.main-content {
+    border: 5px solid red;
+}
+
+.top-content {
+    display: flex;
+    margin-top: 20px;
+    margin-bottom: 20px;
+    flex-direction: row;
+    justify-content: space-between;
+    border: 5px solid lime;
+    /* padding: 20px 20px 20px; */
+}
+
+.contact {
+    width:100%;
+    margin-left: 50px;
+    padding-bottom: 25px;
+    text-align: left;
+}
+
+.Features {
+    margin-right: 20px;
+}
+
+.bottom-content { 
+    display: flex;
+    margin-top: 20px;
+    margin-bottom: 20px;
+    flex-direction: row;
+    justify-content: space-between;
+    border: 5px solid blue;
+    /* padding: 20px 20px 20px; */
+}
+
```

Comment:
these class names are way too specific. If you wanted to use this same layout with different content you would have to start from scratch or use misleading class names. Plus, you are repeating yourself right here. Remember D.R.Y. Don't Repeat Yourself. If 2 elements are going to have the exact same styling, just use the same class name for both instead of copy pasting.